schemas/dag-0/metadata.json

```diff
@@ -685,7 +717,8 @@
     "optionsDataAccessRestriction": {
       "type": "string",
       "enum": [
-        "Closed"
+        "Open – Accessible for researchers of Geosciences",
```

Comment:
Deze enums moeten denk ik ook enumNames worden die mappen op de oude enums totdat we ondersteuning hebben voor nieuwe types access restriction.